README.md

```diff
@@ -71,7 +74,7 @@ Check out the [open issues](https://github.com/KrauseFx/markdown-to-html-github-
 ---
 
 1. Numbered list item 1
-1. Numbered list item 2
+2. Numbered list item 2
```

Comment:
The `1.` is by design, as it allows reordering of list items without having to re-write all the numbers :+1: